tests/foreman/installer/test_installer.py

```diff
@@ -1584,3 +1585,79 @@ def test_installer_timestamp_logs():
 
     :CaseAutomation: NotAutomated
     """
+
+
+def test_installer_inventory_plugin_update(destructive_sat):
+    """DB consistency should not break after enabling the inventory plugin flags
+
+    :id: a2b66d38-e819-428f-9529-23bed398c916
+
+    :steps:
+        1. Enable the cloud inventory plugin flag
+
+    :expectedresults: inventory flag should be updated successfully without any db consistency
+        error.
+
+    :CaseImportance: Medium
+
+    :CaseLevel: System
+
+    :BZ: 1863597
+
+    :customerscenario: true
+
+    """
+    ContentHost(destructive_sat.hostname).create_custom_repos(rhel7=settings.repos.rhel7_repo)
```

Comment:
`destructive_sat` is inherited from the ContentHost would rather have the `create_custom_repos` by default.
`
 destructive_sat.create_custom_repos(rhel7=settings.repos.rhel7_repo)` is not working for you ?

Reply:
yes, you are right, it works.